datahub_sap_hana/ingestion.py

```diff
@@ -113,6 +119,7 @@ def _get_view_lineage_elements(self) -> Dict[Tuple[str, str], List[str]]:
 
         with engine.connect() as conn:
             query_results = conn.execute(LINEAGE_QUERY)
+            logger.debug("Running the query")
```

Comment:
which query, be more specific in the logs
```suggestion
            logger.debug("Lineage query completed")
```